lib/tbot/ca_rotation.go

```diff
@@ -224,11 +224,12 @@ func filterCAEvent(log logrus.FieldLogger, event types.Event, clusterName string
 		)
 	}
 
-	// We want to skip anything that is not host, user, db
+	// We want to skip anything that is not host, user, db, or openssh
 	if !slices.Contains([]string{
 		string(types.HostCA),
 		string(types.UserCA),
 		string(types.DatabaseCA),
+		string(types.OpenSSHCA),
```

Comment:
@capnspacehook This also should be removed. We won't get OpenSSH CA if we're not subscribed for it.

Reply:
Done